### PR TITLE
remove openshift-controller-manager dep on bootstrappolicy

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: pkg/unidling/util/scale.go [hunk 1]
```diff
@@ -22,6 +21,8 @@ import (
 	unidlingapi "github.com/openshift/origin/pkg/unidling/api"
 )
 
+const legacyGroupName = ""
```
Comment (mfojtik) [inline pkg/unidling/util/scale.go]: do we still need this?

Comment (deads2k) [reply pkg/unidling/util/scale.go]: > do we still need this?

Yeah.  Existing data is there and we aren't migrating it.

**Pull request merged**